packages/engine/src/store/types.ts

```diff
@@ -177,17 +182,18 @@ export interface SketchModuleItem {
 
 export type SketchModules = { [key: string]: SketchModuleItem }
 
-export type EnumOption = { value: string | number; label: string }
+export type EnumOption = { value: NodeEnumValue; label: string }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type InputOptions = any
+export type InputOptionNodesConfig = readonly SketchConfigParam[]
 
-export interface Input<Options = InputOptions> {
+export interface Input {
   id: string
   title: string
   type: 'midi' | 'gamepad' | string
```

Comment:
Feel like maybe this should just be string? since there could end up being all sorts of types coming via plugins?

Reply:
I guess type being the way for a plugin to know the input belongs to it (or perhaps that it wants to modify it in the case of say a plugin that mocks midi messages, etc?)